exercise1/exercise1.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
why the `git diff` of first exercise is coming here? I should see only the second exercise here.